kbase-extension/static/kbase/js/widgets/appWidgets2/input/selectInput.js

```diff
@@ -75,103 +66,77 @@ define([
         // DOM EVENTS
 
         function handleChanged() {
-            return {
-                type: 'change',
-                handler: function () {
-                    importControlValue()
-                        .then((value) => {
-                            model.value = value;
-                            channel.emit('changed', {
-                                newValue: value,
+            importControlValue()
```

Comment:
This now gets run when the `'change'` event gets triggered by the select2 widget, instead of being bound to the DOM event.